spec/spec_helper.rb

```diff
@@ -0,0 +1,100 @@
+# rubocop:disable all
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
```

Comment:
While you did a good job working on the project requirements I noticed you disabled rubocop. It is against microverse policy to disable linters because linters help us write clean code. So it would be nice if you enabled rubocop